appserver/extras/payara-micro/payara-micro-boot/src/main/java/fish/payara/micro/boot/PayaraMicroBoot.java

```diff
@@ -437,14 +437,28 @@ public interface PayaraMicroBoot {
      */
     PayaraMicroBoot setMinHttpThreads(int minHttpThreads);
 
-    /**
+  /**
+   * Indicated whether clustering is enabled
```

Comment:
This seems vague - we're not turning off clustering with this option, we're turning off Hazelcast entirely.

```suggestion
   * Indicates whether Hazelcast is enabled.
```

Reply:
Line #441 should read "Indicated whether clustering is enabled". As a matter of fact, FISH-372 requires that, in the new release, the `--nohazelcast` option replaces the `--nocluster` option in the previous release. Given that the role of `--nocluster` was to enable or disable the clustering and that this option is replaced by `--nohazelcast` in the new release, `--nohazelcast` means now "disable or enable the clustering". Hence the comment seems right to me.